docs/blog/newsletter-may-2019.rst

```diff
@@ -0,0 +1,130 @@
+* Collaborate with other teams! UX, Customer Support, Training, and other teams can all be valuable resources for videos.
+* Many community members suggested that docs teams should only commit to video content if they have a dedicated person, or even team, to work on videos.
+
+If you've got any more questions, head on over to `#video <https://writethedocs.slack.com/messages/video/>`_!
+
+------------------------------------------------------
+The variable fortunes of variable placeholders in docs
+------------------------------------------------------
+
+In code examples, you often need to put in placeholders for values that users must provide. But no matter what a hapless writer tries, it seems inevitable that users won't realise they need to substitute in their own data. Here's a summary of the options that our documentarians have provided for their readers:
+
+* Explain the values that must be replaced, followed by the example.
+* Provide an example with placeholder values, then an example with (fake) real values.
+* Format the placeholder text in italic, or write it in all caps.
+* Surround the placeholder text with angle brackets.
+* Beware of indicators that could be taken as part of the code syntax, like angle brackets or curly braces.
+* Don't explain the values to be replaced, because you can assume your audience understands what to do.
+* Implicit in the previous item, but explicit in the discussion: Know Your Audience!
+
+And one person told the story of an API that, if a user copied and pasted the example code without substituting appropriate values, produced this error: "The user your_username is not a valid user. Did you forget to change the sample code to use your own username and password?”
+
+--------------------------------
+Developer to documentarian ratio
+--------------------------------
+
+This month, one community member was looking for ideas for when your company asks "Why aren't our docs as great as <other doc site>?" Especially if you don't have the resources you'd need to make something equivalent.
+
+One approach is to find out how many people the other company puts on docs. For example, many folks find their docs being compared to Stripe's; but community members reported that Stripe's site is custom-built, with dedicated engineers as well as writers (plural, of course).
+
+You can also try breaking down the work that's required to get there, and use that to ask for more people (or for the pressure to be taken off). With positive phrasing: "I'd love to do that! Now, here are the resources we'll need to make it happen."
+
+But how many people *do* you need? The ratio of writers to developers varies hugely. `This report <https://www.infomanagementcenter.com/publications/e-newsletter/cidm-enews-03-17/another-look-at-writer-to-developer-staffing-ratios/>`__ had found that between 1:7 and 1:12 was fairly standard, but community members reported ratios as high as 1:51.
```

Comment:
bit odd to say the report found -- maybe "One survey of articles and forum posts found that..." instead?

Reply:
:+1: